shell/platform/windows/flutter_windows_engine.cc

```diff
@@ -18,6 +18,8 @@
 #include "flutter/shell/platform/windows/task_runner.h"
 #include "third_party/rapidjson/include/rapidjson/document.h"
 
+#undef GetCurrentTime
```

Comment:
Curious how this happened? Do we put this somewhere in the root namespace?

Reply:
This came from one of the windows libraries, I'll add some documentation on it

Reply:
done